lib/elixir/lib/kernel.ex

```diff
@@ -2948,11 +2948,18 @@ defmodule Kernel do
 
       Enum.member?([1, 2, 3], x)
 
+  Elixir also supports a `left not in right` syntax, which evaluates to
```

Comment:
I would say "a" fits better here. Or we could avoid the problem entirely and just say "Elixir also supports `left not in right`..."